core/src/main/java/com/datastax/oss/driver/api/core/metadata/schema/KeyspaceMetadata.java

```diff
@@ -98,7 +98,7 @@ default String describe(boolean pretty) {
         new ScriptBuilder(pretty)
             .append("CREATE KEYSPACE ")
             .append(getName())
-            .append(" WITH replication = { 'class' : '")
+            .append(" WITH REPLICATION = { 'class' : '")
```

Comment:
cqlsh lower-cases it (as well as durable_writes).

Reply:
you are right about that.  Not sure how I got that impression.  Will revert that then.